imagecraft/ubuntu_image.py

```diff
@@ -108,7 +209,7 @@ def ubuntu_image_pack(
 ) -> None:
     """Pack the primed image contents into an image file."""
     cmd: list[str] = [
-        "./ubuntu-image",
+        "ubuntu-image",
```

Comment:
An idea for the future: since we're now depending on ubuntu-image to be installed, could we maybe add a check to see if ubuntu-image is installed and available at the start of imagecraft?

Reply:
I need to learn a bit more about snap dependencies. Because I declared ubuntu-image as a staged-snap so I suppose it is embedded in the imagecraft snap, so I guess it is necessarily present, right?